modules/coremark/Kconfig

```diff
@@ -0,0 +1,86 @@
+#
+# Copyright (c) 2023 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+menuconfig COREMARK
+	bool "Coremark benchmark"
+
+if COREMARK
+
+config COREMARK_ITERATIONS
+	int "Number of the iterations to run"
+	range 10 10000
+	default 1000
+	help
+	  Option is used to specify the number of the iterations to run the benchmark.
+	  Coremark shoud execute at least 10 seconds to have the valid results.
```

Comment:
No, because when we run coremark we don't decide how long it will take. We only give number of iterations the sample must preform. We should set this number high enough that when the sample is executed it should execute for at least 10s.

Reply:
So it's inconsistent with the doc where it's written:
`By default, the iteration quantity is set to the minimum time required for valid results.
For valid results, the test should be executed for at least 10 seconds.`
I see that @annwoj has different suggestion for the doc so maybe it could be used here. At least, please fix the mistake in "should" and add "for" after execute.